### DISCUSSION TITLE
How to display hex colors within README.md?

### DISCUSSION THREAD

**Discussion by thomagos** [community/community#134269] [New to GitHub] [answered]
In the latest issue of the GH Insiders newsletter you show how to display colours in hex on GitHub. This is also covered in the documentation: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#supported-color-models
You can clearly see the expected output, as you can see here in this discussion: `#FF9933` an orange dot behind the hex code.

As written in the docs this does not work inside README.md files. It only displays the hex code without any coloured dot. 
Demo: https://github.com/schwaerzl/color-demo/

Is there **another way** to have something like the feature inside a README?

**Comment by unknownguy49** [comment, marked as answer]
Currently, GitHub does not support displaying colored dots for hex codes directly inside README.md files. However, you can use SVG images to achieve a similar effect. Here's an example:

```markdown
![#FF9933](https://via.placeholder.com/15/FF9933/000000?text=+) `#FF9933`
```

This uses an external image link to display a colored dot.
